src/Analyzers/CSharp/Tests/OrderModifiers/OrderModifiersTests.cs

```diff
@@ -374,6 +375,27 @@ public async Task PartialAtTheEndMethod2()
 @"partial class C
 {
     unsafe partial void M();
+}");
+        }
+
+        [Fact(Skip = "https://github.com/dotnet/roslyn/pull/52297#discussion_r605273814"), Trait(Traits.Feature, Traits.Features.CodeActionsOrderModifiers)]
```

Comment:
Are you okay with keeping the test skipped? @CyrusNajmabadi 

My intention here is to make anyone who looks to fix this later aware of the perf impact.

Reply:
I wouldn't do this, because there's no promise that somebody who tries making this change again will ever find this test given it's just one line in many.

Reply:
@jasonmalinowski Going to delete it if it was decided not to support local functions.

Reply:
can you unskip and just have it test the behavior we decided on?